### ISSUE TITLE
As a billing manager I want to manage annual invoicing on Janeway

### ISSUE THREAD

**Issue by joemull** [BirkbeckCTP/consortial_billing#67]
As a billing manager, I'd like to have a workflow in Janeway that shows me which supporters are due for billing, gives me options for modifying the band if needed, and ideally generates the invoice, though that last part may need to happen elsewhere.
 
- [ ] I can open a page every month and see which supporters need to be billed that month. Generally each supporter is billed once per year, so the list should include all the active supporters for whom I am a billing agent who were last billed at least 11 months ago.
- [ ] I can calculate or manually set a new band for each supporter, based on the current financial year's base bands (which I have previously adjusted for inflation using the tools in #64) and most recent GNI and exchange rates.
- [ ] If no financial year exists in Janeway for the current day, I am warned that I should do an annual update of the kind mentioned in #64 before proceeding with billing.
- [ ] I can generate an email to the supporter with the invoice and send it via Janeway. I can attach the invoice to the email or include it in the email. Perhaps I can also generate the invoice itself, though this needs a bit more user research to figure out how it might integrate with accounting systems.
- [ ] When I send an invoice, a record is logged and kept with the supporter record.
- [ ] I can filter the general supporter list in #66 by whether they've been invoiced in the current financial year.

**Comment by StephDriver** [comment]
TLDR 
Instead of looking at generating invoices, we could focus on collating data for invoices in a easily transferable form (e.g. export to spreadsheet, easily copy individual lines).

Details

From my time working in management accounts in a former life - larger organisations often funnel all invoices through a finance team, such that any other employee 'requests' an invoice, but does not issue it.  This enables the finance team to track monies in and out, chase payments etc.  Some may allow teams to issue their own invoices, and will have templates for this.

While each organisation likely has their own way of doing this - in terms of what software is used to submit invoice requests or  invoice templates - the information included is likely to be similar - so for a billing manager producing a large number of requests, it may be helpful to have an easily accessible summary of that information, e.g. with copy to clipboard buttons, so that they can transfer it across into whatever format they require.
